packages/functions/docs/modules/index.md

```diff
@@ -86,7 +109,7 @@ Returns the location information for the incoming request.
 
 #### Defined in
 
-[packages/functions/src/get-env.ts:1](https://github.com/vercel/vercel/blob/main/packages/functions/src/get-env.ts#L1)
+[packages/functions/src/get-env.ts:6](https://github.com/vercel/vercel/blob/main/packages/functions/src/get-env.ts#L6)
```

Comment:
<!-- Suggestion comments propose improvements to the code. Mention specifically what should be changed and what the benefit would be.  -->
**suggestion (soft-blocking):** use canonical link